.github/workflows/publish.yml

```diff
@@ -0,0 +1,39 @@
+name: Publish artifacts
+on:
+# Will only run when release is published. 
+  pull_request:
+  release:
+    types: 
+      - created
+  workflow_dispatch:
+
+jobs:
+  validate-helm-charts:
```

Comment:
Isn't this requires any building task? Anyone can commit to this repo w/o build?

Reply:
I see, it has only helm charts in this repo.